resenia/src/App.js

```diff
@@ -4,20 +4,17 @@ import { useEffect, useState } from 'react';
 
 function App() {
 
-  const API = 'localhost:8080'
+  const API = 'http://localhost:8080'
 
   const [saludo, setSaludo] = useState("");
 
-  var result = "";
-
   useEffect(() => {
     getHello();
   })
 
   const getHello = () => {
-    fetch('http://localhost:8080/hola')
+    fetch(API + '/hola')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Salgadoema/Grupo-L-012021-frontend/pullRequest?prid=7275486)
```suggestion
    fetch(API + "/hola")
```